src/modules/displaymanager/main.py

```diff
@@ -849,6 +850,8 @@ def set_autologin(self, username, do_autologin, default_desktop_environment):
         de_command = default_desktop_environment.executable
         if os.path.exists(self.os_path("usr/bin/gtkgreet")) and os.path.exists(self.os_path("usr/bin/cage")):
             self.config_data['default_session']['command'] = "cage -d -s -- gtkgreet"
+            if self.greeter_css_location is not None:
+               self.config_data['default_session']['command'] + " -s " + self.greeter_css_location
```

Comment:
I have a couple of questions on this line:
* Shouldn't you be assigning it to something?
* Isn't it `-c`?

Reply:
@dalto8 
1) What do you mean?
2) No. According to `gtkgreet --help:`

`-s, --style              CSS style to use`